frontend/src/Pages/Plantlistpage/PlantListPage.js

```diff
@@ -12,15 +12,23 @@ export default function PlantListPage({ plantList }) {
       <FixedHeader>My Plants</FixedHeader>
       <ItemContainer>
         {plantList.map(({ id, plantname, plantspecies }) => (
-          <PlantCard key={id} name={plantname} species={plantspecies} />
+          <PlantCard
+            key={id}
+            name={plantname}
+            species={plantspecies}
+            onClick={() => openDetail({ id })}
```

Comment:
You could alternatively use the NavLink component of React instead of a onClick Listener